### ISSUE TITLE
Delete collection model includes element IDs which are not unique

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6777]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the model which tells the user that they cannot delete a collection contains element IDs which are not unique and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/my/collections?locale=en and run the Site Improve browser extension

I've included a screenshot of the code snippet flagged by SiteImprove, since this is difficult to see in the UI

![collections_delete](https://github.com/samvera/hyrax/assets/32644254/388e9537-d4c2-4936-8c79-d8c37a341bc3)

**Comment by jlhardes** [comment]
@rjkati I am not seeing this SiteImprove error on dev.nurax.samvera.org but it kind of sounds like it isn't something that will always show on the Collections page in the Dashboard, depending on the collections that you can see. Is this a particular type of collection that ends up having this non-unique <div id>?

I think I need more information on how to get the right type of collections listed (or be the right type of logged in user - possibly a non-admin user?) in order to see this SiteImprove error in action. Are there more details you can provide at this point? I know it has been a while...
